pytorch_toolkit/ote/setup.py

```diff
@@ -1,7 +1,12 @@
+import os
 from setuptools import setup, find_packages
 
+with open(os.path.join(os.path.dirname(__file__), 'requirements.txt')) as read_file:
+    requirements = [requirement for requirement in read_file]
```

Comment:
1.
    ```python
    requirements = list(read_file)
    ```
1. Is it Ok not to strip the lines?

Reply:
1. thanks
2. it works actually, but not documented, so .strip() added